bedrock/thunderbird/templates/thunderbird/email-providers.html

```diff
@@ -0,0 +1,56 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "thunderbird/base-resp.html" %}
+
+{% block page_title %}{{ _('Free Email Providers') }}{% endblock %}
+
+{% block body_id %}email-providers{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'thunderbird-features' %}
+{% endblock %}
+
+{% block content %}
+  <header id="main-feature">
+    <h1>Free Email Providers</h1>
+  </header>
+
+  <main>
```

Comment:
I believe we still do for a11y reasons in IE as IE does not support the main element:
http://caniuse.com/#feat=html5semantic